fw/rbcx-coprocessor/src/ControlLink.cpp

```diff
@@ -57,27 +64,19 @@ void controlUartInit() {
     dmaTxHandle.Init.PeriphDataAlignment = DMA_PDATAALIGN_BYTE;
     dmaTxHandle.Init.Priority = DMA_PRIORITY_MEDIUM;
     HAL_DMA_Init(&dmaTxHandle);
+    HAL_NVIC_SetPriority(controlUartTxDmaIRQn, controlUartTxDmaIrqPrio, 0);
+    HAL_NVIC_EnableIRQ(controlUartTxDmaIRQn);
     LL_USART_EnableDMAReq_TX(controlUart);
 
+    txMessageBuf = xMessageBufferCreate(512);
```

Comment:
Nebylo by lepší konstantu `512` dát do nějaké konstantní proměnné, případně nemusí tohle číslo odpovídat velikosti `rxFifo`?

Reply:
Můžu to vytáhnout. S velikostí rx to nesouvisí, jsou to random čísla

Reply:
V tom případě to asi takhle můžeme nechat.